src/pyramid_debugtoolbar/toolbar.py

```diff
@@ -200,14 +199,22 @@ def toolbar_tween(request):
         except UnicodeDecodeError as e:
             raise URLDecodeError(e.encoding, e.object, e.start, e.end, e.reason)
 
-        last_proxy_addr = None
-        if request.remote_addr:
-            last_proxy_addr = last_proxy(request.remote_addr)
+        client_addr = None
+        if request.remote_addr and ',' not in request.remote_addr:
+            client_addr = request.remote_addr.strip()
+        elif request.remote_addr is not None:
+            warnings.warn(
+                'pyramid_debugtoolbar has detected a broken middleware box '
+                'that modified REMOTE_ADDR with an invalid value and is '
+                'cowardly going to refuse to serve the toolbar. If you run '
+                'into this issue, please PLEASE tell me what server '
+                'software you are running at '
+                'https://github.com/Pylons/pyramid_debugtoolbar/issues/')
```

Comment:
maybe just say "If you see this message, and you think it is incorrect, please open an issue with more details including the proxy you're using and the format of the REMOTE_ADDR"